### PR TITLE
Updates to the Careers pages

### PR DESCRIPTION
**Pull request by sebworks**
Updates to the Careers pages.

## Changes

- Made various content updates as listed in the JIRA task.

## Testing

- Visit `http://localhost:8000/about-us/careers/` and the linked careers pages; verify the content updates listed in the JIRA task. 

## Review

- @anselmbradford 
- @jimmynotjim 
- @schaferjh 

## Screenshots

screen shot 2016-04-21 at 9 37 10 am

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/jinja2/v1/about-us/careers/application-process/index.html [hunk 1]
```diff
@@ -61,13 +61,12 @@
         }}
     </div>
     <section class="job-applications-interfaces">
-        <h2>Different interfaces you may have to use</h2>
+        <h2>What You Will See</h2>
```
Comment (schaferjh) [inline cfgov/jinja2/v1/about-us/careers/application-process/index.html]: Make this sentence case per our other headings -- What you will see

Comment (sebworks) [inline cfgov/jinja2/v1/about-us/careers/application-process/index.html]: Fixed in https://github.com/cfpb/cfgov-refresh/pull/1959/commits/7ff8f47f54c20f54b3a7a9ecd5c602add6a99404

**Comment by schaferjh** [comment]
While we're in here, I just realized this is one of those sections with incorrect page title casing: **All of these headers should be sentence case *not* title case** in the main content of the page.

on /about-us/careers/
Newest Openings 
Job Application Process
Students and Recent Graduates

on /about-us/careers/application-process/
Job Application Process
Current Openings
Asking About your Application
Preparing a Résumé to Apply to a Federal Government Position
Federally Recognized Status and Eligibility for a Position
Federal Ethics Rules and CFPB Ethics Regulations
Equal Employment Opportunity

 on /about-us/careers/students-and-graduates/
Students and Recent Graduates
*program names are fine to leave as title case*

on /about-us/careers/current-openings/
Current Openings

These changes should also be reflected in the blobs the appear at the bottom of the pages -- i.e.
screen shot 2016-04-21 at 9 57 35 am

**Comment by sebworks** [comment]
> While we're in here, I just realized this is one of those sections with incorrect page title casing: All of these headers should be sentence case not title case in the main content of the page.

Fixed in https://github.com/cfpb/cfgov-refresh/pull/1959/commits/bd48764aa18f96dceeaad6061438446b9e70f3f6

**Comment by schaferjh** [comment]
Niiiice. Now, let's sit on this for a second -- I'm verifying a question that has come up between some stakeholders. I will comment when it's cleared to go.

**Comment by jimmynotjim** [comment]
@sebworks you'll need to update the browser tests for the change in title case

**Comment by sebworks** [comment]
>> @sebworks you'll need to update the browser tests for the change in title case

You are correct. Doing that now.

**Comment by KimberlyMunoz** [comment]
Three tests are failing from changing the case, but it looks good and ready to merge after that :+1:

**Comment by schaferjh** [comment]
Just got the 👍  from stakeholders. This is all good from me from a content perspective!

**Comment by jimmynotjim** [comment]
@sebworks Capitalize "Students"

```
Expected [ 'Current openings', 'Job application process', 'Students & recent graduates' ] to equal [ 'Current openings', 'Job application process', 'students & recent graduates' ].
```

**Comment by sebworks** [comment]
>> @sebworks Capitalize "Students"

Fixed in https://github.com/cfpb/cfgov-refresh/pull/1959/commits/96474c3d9e5bf47130e612eb6d816136566a7971

**Comment by jimmynotjim** [comment]
👍  all other errors are unrelated. This is good to merge.

**Pull request merged**